docs/UserGuide.adoc

```diff
@@ -270,12 +270,13 @@ Selects the 1st person in the results of the `find` command
 //tag::bulkimport[]
 === Importing a CSV file : `i`, `import`
 
-Imports a CSV file to the CRM Book. +
+When you have a CSV file of contact list, you can directly import a CSV file to the CRM Book without adding them one by one. +
```

Comment:
This implies that the CSV File contains contacts. A better phrasing would be

Imports a CSV File of Leads to the CRM Book without manually adding the Leads individually.